style.scss

```diff
@@ -0,0 +1,924 @@
+}
+
+header h1 {
+    color: white;
+}
+html, body{
+    /*=====removes visible scroll bar======*/
+    overflow-x: hidden;
+    overflow-y: hidden;
+    width:100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+section {
+    padding-top: 30px;
+    padding-bottom: 30px;
+    width: 70%;
+    z-index: 1;
+    float: left;
+    position: absolute;
+    height:  100%;
+    max-width:35%;
+    /*padding: 100px 0;*/
+    background: #c9dbe9;
+    background: -webkit-linear-gradient(top, #c9dbe9 0%, #fff 100%);
+    background: -linear-gradient(top, #c9dbe9 0%, #fff 100%);
+    background: -moz-linear-gradient(top, #c9dbe9 0%, #fff 100%);
+    border-top-right-radius: 30% 50%;
+    border-bottom-right-radius: 30% 50%;
+}
+
+/*#clouds{*/
+/*    padding: 100px 0;*/
+/*    background: #c9dbe9;*/
+/*    background: -webkit-linear-gradient(top, #c9dbe9 0%, #fff 100%);*/
+/*    background: -linear-gradient(top, #c9dbe9 0%, #fff 100%);*/
+/*    background: -moz-linear-gradient(top, #c9dbe9 0%, #fff 100%);*/
+/*}*/
+
+
+
+/*=========Color Rainbow Boxes========= */
+.box1 {background: #2ecc71;}
+.box2 { background: #f1c40f;}
+.box3 { background: #e67e22;}
+.box4 { background: #ce1126;}
+.box5 { background: #9b59b6;}
+.box6 { background: #3498db;}
+.box7 { background: #1abc9c;}
+/*.box8 { background: #34495e;}*/
+/*.box9 { background: #e743c3;}*/
+/*.box10 { background: #16A085;}*/
+/*=====================================*/
+
+/*=======Navigation Within Rainbow Boxes======*/
+.rightNav{
+    float: right;
+    height: auto;
+    width: 100%;
+    z-index:999999;
+    top: 0;
+    left: 0;
+    background-color: #111;
+    overflow-x: hidden;
+
+}
+.tablinks active{
+    -webkit-animation: bounce;
+}
+.move{
+    justify-content: flex-start;
+}
+
+.rightNav p {
+    z-index: 999999;
+    padding: 15px 8px 6px 8px;
+    text-decoration: none;
+    font-size: 25px;
+    color: #818181;
+    display: block;
+    font-family: 'Share Tech Mono', sans-serif;
+
+}
+
+.container{
+    height: 100vh;
+	/*allow boxes to autoresize with page height*/
+	display: flex;
+	flex-direction: column;
+}
+
+.box{
+    text-align: right;
+    padding: 5px;
+    justify-content: center;
+}
+
+/*allow boxes to autoresize with page height*/
+.container > div {
+  flex: 1;
+}
+
+.box p{
+    color: white;
+    padding-right: 25px;
+}
+
+#tablinkcontainer {
+	float:right;
+	width:100%;
+	transition-duration: 300ms;
+}
+
+/*=================Tabcontent generic style====================*/
+
+.tabcontent {
+	height: 100%;
+    width: 0;
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+	overflow-x: hidden;
+    transition: 0.3s;
+}
+
+.tabcontent::-webkit-scrollbar {
+    display: none;
+}
+
+.overlaycontent {
+	position: relative;
+    top: 25%;
+    width: 100%;
+    text-align: center;
+    margin-top: 30px;
+}
+
+/* =============Start of CSS for Card 1 open page===========*/
+.overlay1  {
+
+    /*background-color: black;*/
+    background-color: #2ecc71;
```

Comment:
So one of the reasons that we use SCSS instead of SCSS is because of the features that SCSS provides. https://sass-lang.com/
SCSS gets built into CSS, so while all of the usual CSS rules work just fine, we can make use of SCSS features to write cleaner, and easier to develop code.

So that we can make this code easier to maintain, we could instead make a common `.overlay` class, which has all of the rules that are common between `.overlay1`, `.overlay2`, etc.
```scss
.overlay {
    a {
    font-size: 36px; // TODO add the rest of the shared layout rules
    }
    .closebtn {
    position: absolute;
    }
}
```
Then, to handle color of each of the tabs, we could keep the `.overlay[1-N]` classes, but this time with a lot of the duplicated rules removed (as they are already set in the `.overlay` class)
```scss
.overlay1 {
    $background-color: #ff00ff; // variables, either in local or global scope, can reduce copy/pastes, and also help define colors only once
    $text-color: #abcabc;
    $highlight-color: #0000ff;
    background-color: $background-color;
    a { color: $highlight-color; }
}
```
This way, we only have to duplicate only a few lines of code for each of the color schemes, instead of a bunch.

Reply:
I agree. Moreover, this strategy is not necessarily special to SCSS or its syntax; it's simply using good effective CSS class structure.

Will continue to rework this stuff, which will likely make it easier to pin down alignment issues of text within the cards.